services/frontend-service/Makefile

```diff
@@ -38,7 +38,7 @@ bin/main: $(GO_FILES) | proto
 GARBAGE += bin/main
 
 run: bin/main
-	KUBERPULT_CDSERVER=localhost:8443 bin/main
+	KUBERPULT_CDSERVER=172.17.0.3:8443 bin/main
```

Comment:
please don't commit this

Reply:
Good point! I've made it possible to specify this on the command line